### ISSUE TITLE
Bug: exponents in paths are broken

### ISSUE THREAD

**Issue by blokhin** [rubenv/point-in-svg-polygon#1]
Hi!
Consider the following path:
M 5.29551E-05 0 L 300 0 L 150 300 z
Then it cannot be determined, whether the point is inside or outside.

**Comment by rubenv** [comment]
There's specific code to handle exponents: https://github.com/rubenv/point-in-svg-polygon/blob/79eefc5736dc2625ad82bcd4172f013b4ed3cac6/lib/index.js#L321-L333

https://github.com/rubenv/point-in-svg-polygon/blob/79eefc5736dc2625ad82bcd4172f013b4ed3cac6/test/segments.js#L57-L62

What failure are you seeing?

**Comment by blokhin** [comment]
How to reproduce:
```
var p = "M 5.29551E-05 0 L 300 0 L 150 200 z";
var segments = splitSegments(p);
var result = isInside([50, 50], segments);
console.log(result);
```
Will output *false*. Then change *5.29551E-05* to *0* and the output will be *true*.

**Comment by rubenv** [comment]
Great, found the issue, this was a bug related to the handling of negative exponents. Thanks a lot for reporting this!

**Comment by rubenv** [comment]
Published as `v1.0.18`.
